webpack.config.js

```diff
@@ -53,18 +54,17 @@ module.exports = async function (_, env) {
     resolveLoader: {
       alias: {
         // async-component-loader returns a wrapper component that waits for the import to load before rendering:
-        async: path.join(__dirname, 'config/async-component-loader')
+        async: path.join(__dirname, 'config', 'async-component-loader')
       }
     },
     module: {
-      // Disable the default JavaScript handling:
-      defaultRules: [],
       rules: [
         {
           oneOf: [
             {
               test: /(\.mjs|\.esm\.js)$/i,
-              type: 'javascript/esm',
+              // don't use strict esm resolution for mjs:
```

Comment:
Why not?

Reply:
Breaks a bunch of stuff in node_modules.

Reply:
Interesting. How come we didn't run into it before?